StardewArchipelago/Archipelago/Gifting/IGiftHandler.cs

```diff
@@ -11,5 +12,9 @@ public interface IGiftHandler
         bool HandleGiftItemCommand(string message);
         void ReceiveAllGiftsTomorrow();
         void ExportAllGifts(string filePath);
+        void SetGiftIDList(List<string> apGifts);

```

Comment:
I think if you do the other changes, none of these 4 will be necessary? Or maybe only 2 out of the 4?